src/lightning_app/core/work.py

```diff
@@ -316,9 +316,16 @@ def num_successes(self) -> int:
 
         return has_succeeded_counter
 
+    def _get_property_if_exists(self, name: str) -> Union[property, None]:
+        return getattr(self.__class__, name, None)
+
     def __setattr__(self, name: str, value: Any) -> None:
-        setattr_fn = getattr(self, "_setattr_replacement", None) or self._default_setattr
-        setattr_fn(name, value)
+        propObject = self._get_property_if_exists(name)
```

Comment:
```suggestion
        property_object = self._get_property_if_exists(name)
```
let's follow Pythons standard naming scheme :)

Reply:
Modified this in [72be4ea](https://github.com/Lightning-AI/lightning/pull/14259/commits/72be4ea69d961eda6d16eae6e4e3f77eddb699f6)